.github/workflows/gh-pages.yml

```diff
@@ -28,6 +28,10 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install -r requirements.txt -r requirements-dev.txt
           python -m pip install '.[all]'
+      - uses: actions/cache@v4
+        with:
+          path: docs/_build
+          key: ${{ runner.os }}-sphinx
```

Comment:
At the first run, no cache was found. But according to [this page](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/caching-dependencies-to-speed-up-workflows#using-the-cache-action), a new cache should be created if the job runs successfully. 

> On a cache miss, the action automatically creates a new cache if the job completes successfully

![image](https://github.com/user-attachments/assets/eea48076-96fa-493b-99ed-5e8cf76beca2)